package.json

```diff
@@ -31,14 +31,21 @@
 		"eslint-plugin-react-hooks": "^4.6.0",
 		"eslint-plugin-react-refresh": "^0.4.3",
 		"jsdom": "^22.1.0",
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0",
```

Comment:
Originally we planned to ship them as regular dependencies, then Fabio encountered issues when used in react-v17 app (Webchat). 

Now those a peer-deps with loose versions for production build, and another is for us.

@fabclj We should lower it to 17, to avoid any potential problems in our dependency graph. Also for us to avoid the v18+ react API

Reply:
It was required. using react 18 on dependecies was a blocker for my task. I faced the problem where 2 react instances are running. One from the webchat (17) and one from the chat components (18). This is going to create conflict as soon as you start using hooks, for example.
You can test by yourself buy deploying webchat v3 and run `npm ls react` and `npm ls react-dom`.

What I did is a common solution (not the only one) many libraries are also doing in order to avoid this issue. Also externalizing react on vite config has to be done in our case.
Potentially we can also move react-player to devDeps and peerDeps, but need to check the compatibilities. I believe we need to handle this kind of set-up related topic on a different PR, here the scope is just to build the components.

@kwinto I also tough about it, but I din't continue because it requires some compatibility check. We need to be sure all devDeps are still fine with the react downgrade. It not, need to align the one are complaining. I would address this in a new PR where we can focus on this topic, as I said before.

Reply:
sounds good to me